tests/integration/values.rs

```diff
@@ -7,22 +7,33 @@ use assert_cmd::prelude::*;
 use indoc::indoc;
 
 #[test]
-fn it_accepts_template_values_file() {
+fn it_accepts_template_values_from_multiple_places() {
     let template = tmp_dir()
+        .file(
+            "my-env-values.toml",
+            indoc! {r#"
+                [values]
+                MY_VALUE1 = "env-file-value"
```

Comment:
question: could you also devote a dedicated section in the README.md to document this behaviour?

especially important for template provider to highlight is the fact that the variable names should be UPPER CASED to be able to be feed via env variables.